routes/runtime/github.rb

```diff
@@ -22,24 +22,26 @@ class Clover
       # caches by scope, pushing entries for later scopes to the end of the
```

Comment:
We need to update this comment to match with the query approach.

Reply:
I'll take care of that as well.